terms/demographics/ageOfDeathUnits.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-demographics.ageOfDeathUnits-0.0.1",
+  "type": "string",
+  "description": "The death age unit of measure (year or gestational week)"
```

Comment:
Can we set it so that if it's human we have year and gestational week, and if animal model months. I'm concerned if we have both months and year for human someone may make a mistake where 18 yrs end up as 18 months

Reply:
We could specify terms such as "Human - Year", "Human - Gestational Week", "Animal Model - Month", but there is currently no way to limit the enumerated list of one term based on another term (species).  We would have to come up with two ageOfDeathUnits keys in order to do this - one human-specific for the Individual Human template and the other animal-specific for the Individual Animal template.

Reply:
I think the concern of putting 'months' depends on how people are filling these out. If they are using a neat system where there are auto-populated drop-down boxes, then I could see this being an issue since it's easy to select the wrong one. However, in that case, mistakes would be an issue for all drop-downs, not just 'ageOfDeathUnits'. If they are hand-typing or using a program to generate metadata (i.e. using a deliberate, hands-on method), this is much less likely to be a mistake that would be made.

Reply:
We will need months for mice. This isn't quite correct since for primate models year would probably be preferable ("Human - Year", "Human - Gestational Week", "Animal Model - Month",)